src/train.jl

```diff
@@ -89,3 +89,26 @@ function sciml_train(loss, θ, opt::Optim.AbstractOptimizer;
            Optim.Options(extended_trace=true,callback = _cb,
                          f_calls_limit = maxiters))
 end
+
+function sciml_train(loss, lower_bounds, upper_bounds, θ, opt::Optim.AbstractConstrainedOptimizer;

```

Comment:
move lower and upper bounds to required kwargs. I never thought this was a good idea by Optim